certdb/pg/migrations/002_AddMetadataToCertificates.sql

```diff
@@ -0,0 +1,21 @@
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE certificates
+ADD COLUMN issued_at timestamptz,
+    ADD COLUMN not_before timestamptz,
+    ADD COLUMN originating_host bytea,
+    ADD COLUMN sans bytea,
+    ADD COLUMN tags bytea,
+    ADD COLUMN common_name bytea,
+    ADD COLUMN filename bytea,
+    ADD COLUMN application_name bytea;
```

Comment:
Maybe we should start the process of not using `bytea` type where it doesn't make sense, starting with these new columns.

Reply:
Yeah, I'll update the pg + mysql migrations to be `TEXT`

Reply:
done